spec/vehicle_locations.schema.json

```diff
@@ -0,0 +1,189 @@
+      "title": "ID referencing stops.stop_id",
+      "description": "Identifies the stop the vehicle is approaching or serving currently. References GTFS"
+    },
+    {
+      "name": "current_status",
+      "type": "string",
+      "description": "Indicates the status of the vehicle in reference to a stop. References GTFS-Realtime",
+      "constraints": {
+        "enum": [
+          "Incoming at",
+          "Stopped at",
+          "In transit to"
+        ]
+      }
+    },
+    {
+      "name": "latitude",
+      "type": "number",
+      "description": "Degrees North, in the WGS-84 coordinate system. References GTFS-Realtime"
+    },
+    {
+      "name": "longitude",
+      "type": "number",
+      "description": "Degrees East, in the WGS-84 coordinate system. References GTFS-Realtime"
+    },
+    {
+      "name": "gps_quality",
+      "type": "string",
+      "description": "Indicates the quality of data and communication provided by the GPS.",
+      "constraints": {
+        "enum": [
+          "Excellent",
+          "Good",
+          "Poor"
+        ]
+      }
+    },
+    {
+      "name": "heading",
+      "type": "number",
+      "description": "Heading, in degrees, clockwise from true north, e.g., 0 would mean north and 90 would mean east. This can be the compass bearing, or the direction towards the next stop or intermediate location. This should not be deduced from the sequence of previous positions, which clients can compute from previous data. References GTFS-Realtime"
+    },
+    {
+      "name": "speed",
+      "type": "number",
+      "description": "Momentary speed measured by the vehicle, in meters per second. References GTFS-Realtime"
+    },
+    {
+      "name": "odometer",
+      "type": "integer",
+      "description": "Odometer value, in meters. References GTFS-Realtime"
+    },
+    {
+      "name": "schedule_deviation",
+      "type": "integer",
+      "description": "Indicates schedule adherence in seconds. A negative value represents an early vehicle. (An unscheduled trip would not have a scheduled deviation.)"
+    },
+    {
+      "name": "headway_deviation",
+      "type": "integer",
+      "description": "Indicates headway adherence in seconds. A negative value represents a shorter than scheduled headway."
+    },
+    {
+      "name": "in_service",
+      "type": "string",
+      "description": "Indicates status of travel with regard to service.",
+      "constraints": {
+        "enum": [
+          "In passenger service",
+          "En-route to service",
+          "Traveling from a service trip",
+          "On a layover",
+          "Returning to a garage due to a suspension or breakdown",
+          "Travel for on-route replacement of breakdown",
+          "Other not in service"
+        ]
+      }
+    },
+    {
+      "name": "schedule_relationship",
+      "type": "string",
+      "description": "Indicates the status of the stop. ",
+      "constraints": {
+        "enum": [
```

Comment:
Referencing https://gtfs.org/realtime/reference/#enum-schedulerelationship (noting since `unscheduled` is still experimental).

We'll probably want to revisit styling choices here in the future, namely to align across references.

Reply:
Add "references GTFS-realtime VehiclePosition.schedule_relationship" to the description key, or just "GTFS-realtime".